RayLin.md

```diff
@@ -1,14 +1,13 @@
-# 10/24 ~ 10/28
+# 11/01 ~ 11/02
 
 - [Video Control]
   - Bug 1271765 - Visual refresh of media controls
-    - r+dholbert
+    - landing
```

Comment:
Sorry, it was landing then. Now I'm waiting for a few intermittent failed retry, if passed, I'll flag new `checked-needed`.